sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/Analytics.java

```diff
@@ -168,6 +168,8 @@ static synchronized void unsetInstance() {
 
     /**
      * Enable or disable Analytics service.
+     * <p>
+     * The state is stored on disk, so it won't change in following app launches.
```

Comment:
Disk is probably not the best term for phone storage.

The state is persisted if the application process restarts or something like that should do the trick without to mention disk.